LokiStart_release_v3.0_connor_modify.ps1

```diff
@@ -0,0 +1,153 @@
+# LokiStart V3.0
+$LokiStart_Verison_Tag = 'V3.0'
+$Description = 'Change download directory into C:\ti\ti-cc2340'
+
+$origin_director = pwd
+
+#############################################################################################################################################################################################
+########################################################### [Do not change] URL and NAME of the Softwares ###################################################################################
+#############################################################################################################################################################################################
+$ccs_version = '12.4.0'
+############################### Get latest sdk version through gihub ########################################################################################################################
+$sdk_product_json_url = "https://raw.githubusercontent.com/TexasInstruments/simplelink-lowpower-f3-sdk/main/.metadata/product.json"
+$sdk_product_json_file = Invoke-RestMethod -Uri $sdk_product_json_url
+$sdk_version = $sdk_product_json_file.version
+
+$compiler_verison = '2.1.3.LTS' #'2.1.2.LTS'
+$rtos_version = '202104.00'
+
+$ccs_name = 'CCS12.4.0.00007_win64'
+$sdk_line_name = $sdk_version -replace '\.','_'
+$sdk_name = 'simplelink_lowpower_f3_sdk_' + $sdk_line_name #'simplelink_lowpower_f3_sdk_7_10_00_35'
+$compiler_name = 'ti_cgt_armllvm_2.1.3.LTS_windows-x64_installer' #'ti_cgt_armllvm_2.1.2.LTS_windows-x64_installer'
+$rtos_name = 'FreeRTOSv202104.00'
+
+$ccs_url = 'https://dr-download.ti.com/software-development/ide-configuration-compiler-or-debugger/MD-J1VdearkvK/'+ $ccs_version + '/' + $ccs_name + '.zip'
+$sdk_url = 'https://dr-download.ti.com/software-development/software-development-kit-sdk/MD-emMPuXshOG/' + $sdk_version + '/' + $sdk_name + '.exe'
+$compiler_url = 'https://dr-download.ti.com/software-development/ide-configuration-compiler-or-debugger/MD-ayxs93eZNN/' + $compiler_verison + '/' + $compiler_name + '.exe'
+$rtos_url = 'https://github.com/FreeRTOS/FreeRTOS/releases/download/' + $rtos_version + '/' + $rtos_name + '.zip'

```

Comment:
FreeRTOS is one of the components of SDK after version 7.40. So you should remove the FreeRTOS part.